components/Table/style.module.scss

```diff
@@ -301,11 +301,12 @@ $cell-height: 49px;
   }
 }
 
-/* Headless Table */
+/* Headerless Table */
 .table {
   border-collapse: collapse;
 
-  :not(thead tbody) tbody:first-child {
+  :not(thead tbody) tbody:first-child td {
+    
```

Comment:
Just remove the extra line break and I think we are good!

Reply:
Did you investigate the border-collapse need? I see it still there

Reply:
border-collapse needs to be there if not it stacks the borders.

Reply:
I think rather than requiring it, we need to work around it. It affects the size of the cells, so we cannot use it. Where is the stacking occurring and under what conditions?